.gitignore

```diff
@@ -1 +1,79 @@
-.DS_Store
+.### OSX ###
```

Comment:
Some of this looks a bit unnecessary lol

Reply:
Oh lol, yeah that was autogenerated by .vscode